src/plugins/Trader/types.ts

```diff
@@ -0,0 +1,98 @@
+export interface Settings {
+    currency: Currency
+}
+
+export enum Platform {
```

Comment:
```suggestion
export const enum Platform {
```

Reply:
No, const enum is not `transpileModule` safe.